Cargo.toml

```diff
@@ -43,3 +44,7 @@ path = "examples/box_game/box_game_spectator.rs"
 [[example]]
 name = "box_game_synctest"
 path = "examples/box_game/box_game_synctest.rs"
+
+[[example]]
```

Comment:
It would be nice to add a short blurb in the `README.md` explaining how to run this stress test:

```bash
cargo run --release --example particles -- --local-port 7000 --players localhost 127.0.0.1:7001 --input-delay 0 --desync-detection-interval 1 --rate 1
cargo run --release --example particles -- --local-port 7001 --players 127.0.0.1:7000 localhost --input-delay 0 --desync-detection-interval 1 --rate 1
```

Reply:
I put in a clap help section instead, which I think makes it easy to discover (and keep in sync, since it's in the source file).

```
dev/bevy_ggrs particles-stress-test↑1 ~1 →1 …1 cargo run --release --example particles
    Finished release [optimized] target(s) in 0.31s
     Running `target\release\examples\particles.exe`
error: the following required arguments were not provided:
  --local-port <LOCAL_PORT>

Usage: particles.exe --local-port <LOCAL_PORT>

For more information, try '--help'.
error: process didn't exit successfully: `target\release\examples\particles.exe` (exit code: 2)
dev/bevy_ggrs particles-stress-test↑1 ~1 →1 …1 cargo run --release --example particles -- --help
    Finished release [optimized] target(s) in 0.30s
     Running `target\release\examples\particles.exe --help`
Stress test for bevy_ggrs

## Basic usage:

Player 1:

cargo run --release --example particles -- --local-port 7000 --players localhost 127.0.0.1:7001

Player 2:

cargo run --release --example particles -- --local-port 7001 --players 127.0.0.1:7001 localhost

Usage: particles.exe [OPTIONS] --local-port <LOCAL_PORT>

Options:
  -l, --local-port <LOCAL_PORT>
          The udp port to bind to for this peer

  -p, --players <PLAYERS>...
          Address and port for the players. Order is significant. Put yourself as "localhost".

          e.g. `--players localhost 127.0.0.1:7001`

  -s, --spectators <SPECTATORS>...
          Address and port for any spectators

  -i, --input-delay <INPUT_DELAY>
          How long inputs should be kept before they are deployed. A low value, such as 0 will result in low latency, but plenty of rollbacks

          [default: 2]

  -d, --desync-detection-interval <DESYNC_DETECTION_INTERVAL>
          How often the clients should exchange and compare checkums of state

          [default: 10]

      --continue-after-desync
          Whether to continue after a detected desync, the default is to panic

  -n, --rate <RATE>
          How many particles to spawn per frame

          [default: 100]

  -f, --fps <FPS>
          Simulation frame rate

          [default: 60]

      --max-prediction <MAX_PREDICTION>
          How far ahead we should simulate when we don't get any input from a player

          [default: 8]

      --reflect
          Whether to use reflect-based rollback. This is much slower than the default clone/copy-based rollback

  -h, --help
          Print help (see a summary with '-h')
dev/bevy_ggrs particles-stress-test↑1 ~1 →1 …1
```